src/components/ContentSidebar/ActivityFeed/icons/IconActivityFeedEmptyState.js

```diff
@@ -0,0 +1,140 @@
+/**
+ * @flow
+ * @file IconActivityFeedEmptyState icon component
+ */
+
+import React, { ReactNode } from 'react';
+
+import AccessibleSVG from 'box-react-ui/lib/icons/accessible-svg';
+
+type Props = {
+    className: string,
+    width: number,
+    height: number
+};
+
+const IconActivityFeedEmptyState = ({ className = '', width = 90, height = 90 }: Props): ReactNode => (
```

Comment:
All icons should come from box-react-ui, so this file should be moved there unless its already there.